aiida/cmdline/commands/cmd_export.py

```diff
@@ -79,10 +79,11 @@ def inspect(archive, version, data, meta_data):
     show_default=True,
     help='Include or exclude comments for node(s) in export. (Will also export extra users who commented).'
 )
+@click.option('--debug', is_flag=True, default=False, flag_value=True, help='Show debug messages.', hidden=True)
```

Comment:
I suggest using the existing `params.options.VERBOSE` option instead (also since the extra print statements are not excessive).

Reply:
I don't like this, since I consider the extra print statements a developer/debug thing. Not for the common user. "Verbose" smells more like a valid flag a user might use, where "debug" is clearly _only_ for developers, or people who are interested in the inner workings.

Reply:
In the end, I consider the progress bar (including summary/header) to be a complete exchange for the current print statements. The old print statements are still useful for debugging however, since they are static in the output and show up at given places.

Reply:
Ok. In that case I suggest you add it in params.options.DEBUG so that other commands can reuse it (e.g. the restapi one probably should)